src/AppBundle/Controller/APIController.php

```diff
@@ -292,14 +292,58 @@ public function getPostAction(Request $request, $post_id)
         // throw an error.
         if (!$post) {
             throw $this->createNotFoundException(
-                'No post found for id ' . $id
+                'No post found for id ' //. $id
             );
         }
         
         $serializer = $this->container->get('serializer');
         $reports = $serializer->serialize($post, 'json');
         return new Response($reports);
     }
+      /**
+     * @Route("/posts/remove", name="api_remove_post")
```

Comment:
Change the route to a DELETE request to /posts. Not a POST request to /posts/remove